cypress/pages/loginPage.js

```diff
@@ -0,0 +1,28 @@
+class homeSaucePage {
+    elements = {
+        usernameInput: () => cy.get("#user-name"),
+        passwordInput: () => cy.get("#password"),
+        loginBtn: () => cy.get("#login-button"),
+        errorMessage: () => cy.get('h3[data-test="error"]'),
+    };
+
+    typeUsername(username) {
+        this.elements.usernameInput().type(username);
+    }
+
+    typePassword(password) {
+        this.elements.passwordInput().type(password);
+    }
+
+    clickLogin() {
+        this.elements.loginBtn().click();
+    }
+
+    submitLogin(username, password) {
+        this.elements.usernameInput().type(username);
+        this.elements.passwordInput().type(password);
+        this.elements.loginBtn().click();
+    }
```

Comment:
if you split your login steps like:
- insert username
- insert password
- click button
you will not need this one